pom.xml

```diff
@@ -28,7 +28,7 @@
     <parent>
         <groupId>org.jboss</groupId>
         <artifactId>jboss-parent</artifactId>
-        <version>21</version>
+        <version>11</version>
```

Comment:
please use version 22, it has important changes wrt to building on JDK9.
It is what all projects are moving to, WildFly master & upcoming EAP 7.1 uses that.